alerter.c

```diff
@@ -3,18 +3,40 @@
 
 int alertFailureCount = 0;
 
-int networkAlertStub(float celcius) {
+enum environment
+{
+    Test,
+    Prod
+};
+
+
+int networkAlerterReal(float celcius){
+    // this method will connect with real devices and decides based on device and input temp
+    // for now returning 200
     printf("ALERT: Temperature is %.1f celcius.\n", celcius);
-    // Return 200 for ok
-    // Return 500 for not-ok
+    return 200;
+}
+
+int networkAlertStub(float celcius){
+    printf("ALERT: Temperature is %.1f celcius.\n", celcius);
+    // Return 200 for ok, ok is celcius < 800
+    // Return 500 for not-ok, not-ok is celcius >= 800
     // stub always succeeds and returns 200
     return 200;
 }
 
-void alertInCelcius(float farenheit) {
+int networkAlerter(enum environment env, float celcius){
+    if(env == 0) return networkAlertStub(celcius);

```

Comment:
This will keep the stub even in the production binary. Is there a way to exclude it during production?